t/14-isotovideo.t

```diff
@@ -51,6 +52,20 @@ subtest 'get the version number' => sub {
     ok(!-e bmwqemu::STATE_FILE, 'no state file was written');
 };
 
+subtest 'color output can be configured via the command-line' => sub {
+    chdir($pool_dir);
+    unlink('vars.json') if -e 'vars.json';
+    my $out = stderr_from { isotovideo(opts => "--color=yes casedir=$data_dir/tests schedule=foo,bar/baz _exit_after_schedule=1") };
+    isnt($out, colorstrip($out), 'logs use colors when requested');
+    $out = stderr_from { isotovideo(opts => "--color=no casedir=$data_dir/tests schedule=foo,bar/baz _exit_after_schedule=1") };
+    is($out, colorstrip($out), 'no colors in logs');
+  TODO: {
+        local $TODO = 'STDERR and STDOUT are always absent here';
+        $out = stderr_from { isotovideo(opts => "--color=auto casedir=$data_dir/tests schedule=foo,bar/baz _exit_after_schedule=1") };
+        isnt($out, colorstrip($out), 'logs use colors by default');
+    }
```

Comment:
I don't feel like the TODO sections in tests work out for us. We should only merge changes without new TODO blocks in tests.